src/styles/styles.scss

```diff
@@ -0,0 +1,43 @@
+.box {
+  display: grid;
+  grid-gap: 0;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-auto-rows: auto;
+  font-size: 100px;
+  color: #fff;
+
+  @media (min-width: 900px) {
+    grid-template-columns: repeat(3, minmax(300px, 1fr));
+  }
+}
+
+.box__item {
+  display: grid;
+  height: 300px;
+  align-items: center;
+  justify-content: center;
+}
+
+.box__item--color-1 {
+  background-color: #f00;
+}
+
+.box__item--color-2 {
+  background-color: #c00;
+}
+
+.box__item--color-3 {
+  background-color: #900;
+}
+
+.box__item--color-4 {
+  background-color: #600;
+}
+
+.box__item--color-5 {
+  background-color: #300;
+}
+
+.box__item--color-6 {
+  background-color: #000;
+}
```

Comment:
use @each to assign background color to each box (you event don't need a modifier for this)